### ISSUE TITLE
Fix `truffle watch`

### ISSUE THREAD

**Issue by cgewecke** [trufflesuite/truffle#1150]
```
TypeError: fsevents is not a constructor
    at createFSEventsInstance 
```

On OSX, possibly others, originating in `chokidar`

**Comment by gnidan** [comment]
Proposal: we just remove this command (#1151 does this), and instead, we add a `--watch` flag:

- `truffle compile --watch` to recompile on changes
- `truffle test --watch` to rerun tests on changes

WDYT?

**Comment by cgewecke** [comment]
@gnidan Yes sounds good.

**Comment by ferittuncer** [comment]
Doesn't work on Ubuntu either.

**Comment by gnidan** [comment]
cc @eggplantzzz

**Comment by CruzMolina** [comment]
@eggplantzzz is this fixed in #1305?

**Comment by eggplantzzz** [comment]
I think it should be fixed for v5 as #1305 is now merged into next.
